src/reduce.test.js

```diff
@@ -11,9 +11,42 @@ describe('reduce', () => {
     delete Array.prototype.reduce2;
   });
 
-  it('should ', () => {
+  it(`should be declared`, () => {
+    expect(Array.prototype.reduce2).toBeInstanceOf(Function);
+  });
+
+  it(`should be correctly if 'startValue' === 0`, () => {
```

Comment:
The test description `should be correctly if 'startValue' === 0` could be improved for clarity. Consider rephrasing it to something like `should return the correct result when 'startValue' is 0`.